chainstate-storage/src/store.rs

```diff
@@ -412,7 +412,9 @@ pub(crate) mod test {
     use common::chain::{Destination, OutputPurpose, TxOutput};
     use common::primitives::{Amount, H256};
     use crypto::key::{KeyKind, PrivateKey};
-    use crypto::random::{make_pseudo_rng, Rng};
+    use crypto::random::Rng;
+    use rstest::*;
+    use test_utils::random::*;
```

Comment:
I'm somewhat OK with wildcard imports in tests, but I'm not sure it is justified here. Especially for the external library (`rstest`). We only need to import two attributes, right?..